addons/website/static/src/snippets/s_mega_menu_images_subtitles/000.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module **/
+
+import publicWidget from 'web.public.widget';
+
+publicWidget.registry.MegaMenuImagesSubtitles = publicWidget.Widget.extend({
+    selector: ".s_mega_menu_images_subtitles",
+
+    /**
+     * @override
+     */
+    start() {
+        // TODO: remove in master
```

Comment:
@aans-odoo As we have added this Public Widget specifically to handle case for people who already dropped the snippet so it please add note on the file, we will remove whole file in future.

Reply:
I've made the suggested changes, thanks.